streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -735,11 +730,11 @@ public void close() {
     }
 
     private double megabytesPerSec(long time, long processedBytes) {
-        return  ((double) processedBytes / 1024 / 1024) / (time / 1000.0);
+        return  ((double) processedBytes / 1024.0 / 1024.0) / (time / 1000.0);
```

Comment:
you can remove the cast to `double` now that you are using `1024.0` same below on line 737